src/App.test.tsx

```diff
@@ -0,0 +1,26 @@
+
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import App from './App';
+
+test('renders Hello World title', () => {
+  render(<App />);
+  const titleElement = screen.getByText(/Hello World!/i);
+  expect(titleElement).toBeInTheDocument();
+});
+
+test('renders Primary and Dashed buttons', () => {
+  render(<App />);
+  const primaryButton = screen.getByText(/Primary Button/i);
+  const dashedButton = screen.getByText(/Dashed Button/i);
+  expect(primaryButton).toBeInTheDocument();
+  expect(dashedButton).toBeInTheDocument();
+});
+
+
+
+test('simple test', () => {
```

Comment:
I think we don't need it. Understood that you used it to test your setup but now it can be removed as it don't test our actual application